migrations/template.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function runMigration(migration) {
```

Comment:
Do we still need this file? Since all the boilerplate is gone, this seems superfluous.